setup.py

```diff
@@ -1,43 +1,101 @@
+import os
+import re
 from setuptools import setup
-from mu import __version__
 
 
-with open('README.rst') as f:
+base_dir = os.path.dirname(__file__)
+
+
+DUNDER_ASSIGN_RE = re.compile(r"""^__\w+__\s*=\s*['"].+['"]$""")
+about = {}
+with open(os.path.join(base_dir, 'mu', '__init__.py'), encoding='utf8') as f:
+    for line in f:
+        if DUNDER_ASSIGN_RE.search(line):
+            exec(line, about)
```

Comment:
Rather than doing this, would it be possible to move the locale stuff to a different file to keep the __init__.py file free of dependency imports?

Reply:
AFAICT, unfortunately not, due to the nature of `gettext.translation` that, when run, installs the pseudo-builtin `_` functions, use throughout the code. So it needs to be called *very early on*...

PS: To get a feeling of the impact, try commenting out the two locale-setup related lines from `__init.py__` and launch mu. :-)

PPS: Why remove imports from `__init__.py`, by the way? To avoid this code in `setup.py`? IMHO, it is never a good idea to import the module/package to be distributed in there.

Reply:
Even more PS: Thanks for taking a look at this! :)